futurenhs.app/UI/scss/modules/card/_card.scss

```diff
@@ -61,6 +61,10 @@ $card-hero-padding: 10px;
         margin-bottom: $nhsuk-spacing-point-2;
         max-width: 60%;
```

Comment:
Deleting this - max-width: 60% will have the same effect but for all cards which would be much more generic and make the change everywhere